src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
-// write code here
+// Working with data
+// Making array from tag nodes
+const dataList = Array.from(document.querySelectorAll('li'));
+
+//  Function to convert USD to numbers
+function convertation(arg) {
+  return parseFloat(arg.replace(/[$,]/g, ''));
+}
+
+// Function getEmployees
+function getEmployeesConvertation(data) {
+  const listOfEmployees = dataList.map((item) => ({
+    name: item.textContent.trim(),
+    position: item.dataset.position,
+    salary: item.dataset.salary,
```

Comment:
The `salary` should be converted to a number using the `convertation` function. Consider using `salary: convertation(item.dataset.salary)`.